helm/cert-manager-app/values.yaml

```diff
@@ -257,6 +257,15 @@ global:
     # global.securityContext.runAsNonRoot
     runAsNonRoot: true
 
+    seccompProfile:
+      type: RuntimeDefault
+
+  containerSecurityContext:
+    allowPrivilegeEscalation: false
```

Comment:
isn't `readOnlyRootFilesystem: true` missing here?

Reply:
The securityContext of a Pod overrides fields set in containerSecurityContext

[SecurityContext holds security configuration that will be applied to a container. Some fields are present in both SecurityContext and PodSecurityContext. When both are set, the values in SecurityContext take precedence.](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#podsecuritycontext-v1-core)